AdventOfCode.Library/Day11/Part2.fs

```diff
@@ -0,0 +1,50 @@
+module AdventOfCode.Library.Day11.Part2
+
+open AdventOfCode.Library.Common
+open AdventOfCode.Library.Day11.Part1
+
+let simulateRound (monkeys: Monkey array) : Monkey array =
+    let m = Seq.fold (*) 1 (Seq.map (fun monkey -> monkey.Divisor) monkeys)
+
+    for monkey in monkeys do
+        for l1 in monkey.Items do
+            let n =
+                match monkey.Operand with
+                | Old -> l1
+                | Value x -> x
+
+            let l2 =
+                match monkey.Operation with
+                | Add -> int64 l1 + int64 n
+                | Multiply -> int64 l1 * int64 n
+
+            let l3 = int (l2 % int64 m)
```

Comment:
I got the modulo trick early on, but ended up getting the wrong answer due to overflow. Figuring out how to juggle `int` and `int64` took me a minute.